### PR TITLE
Added required dark theme 

### PR DESCRIPTION
**Pull request by A-nirvana**
I've added the said dark theme and changed some other component dimensions
Screenshot (23)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -238,8 +238,11 @@ textarea.noteContent {
   padding: 5px 0;
 }
 
+#image{
+  animation: fadein 1.2s;
```
Comment (arpitghura) [inline style.css]: Decrease the animation time to 700ms.

File: main.js [hunk 2]
```diff
@@ -166,7 +173,14 @@ darkBtn.addEventListener("click",()=>{
...
     const textParts = document.querySelectorAll(".noteContent")
     const headings = document.querySelectorAll(".inputHeading")
+    if(mainWindow.classList.value=="dark-mode"){
+        document.getElementById("image").src = "no-notes.gif"
+    }
+    else{
+        document.getElementById("image").src = "no-notes-dark.gif";
+    }
     mainWindow.classList.toggle("dark-mode");
+    console.log(mainWindow.classList.value)
```
Comment (arpitghura) [inline main.js]: Remove console logs

**Comment by A-nirvana** [comment]
@arpitghura please give it a look, I have completed my assignment. Tell me if there's anything else you'd like me to add.x
Screenshot (25)

File: style.css [hunk 3]
```diff
@@ -144,14 +150,29 @@ body {
 }
 
 .note {
+  animation: fadein 3s;
```
Comment (arpitghura) [inline style.css]: Decrease this animation time to `700 ms`

File: logo-dark.jpg
Comment (arpitghura) [inline logo-dark.jpg:1]: Remove this file.

**Review by arpitghura — CHANGES_REQUESTED** [review]
You did great work, but navbar does not look great in the dark mode. So, it's better to not apply dark mode to the header.

Previously, I said yes to make changes but this does not look great.

So, undo the last commit you made and resolve these issues.

**Comment by A-nirvana** [comment]
@arpitghura I have made the changes that you requested. If this issue has been resolved then I'd like to work on other issues in the app as well. .Else please tell me the necessary edits to make. Thanks.

File: style.css (hunk 3 above)
Comment (arpitghura) [reply style.css]: Please resolve this.

File: style.css (hunk 1 above)
Comment (arpitghura) [reply style.css]: Do these.

File: style.css [hunk 4]
```diff
@@ -144,14 +144,29 @@ body {
 }
 
 .note {
+  animation: fadein 0.7s;
```
Comment (arpitghura) [inline style.css]:
```suggestion
  animation: fadein 0.3s;
```
Decrease this to 0.3 seconds as this speed up the total time.

**Review by arpitghura — CHANGES_REQUESTED** [review]
Do this change and rest all is good.

File: style.css (hunk 4 above)
Comment (A-nirvana) [reply style.css]: I've completed the task. I hope its all good now.

**Comment by A-nirvana** [comment]
@arpitghura Please review this and tell me if I can start working on other issues as well.

**Comment by A-nirvana** [comment]
@arpitghura you didn't mention the level of this issue.  If inferred from the "Advanced" tag, it should be a level 4 issue, or even if its a level 2 then please add the "level 4" or a "level 2" tag to it. This would be very beneficial to me. Thanks, pardon my audacity.

**Review by arpitghura — APPROVED** [review]
Looks Good

**Comment by A-nirvana** [comment]
close the issue?? It'll be helpful

**Pull request merged**